wdl-ast/src/v1/workflow.rs

```diff
@@ -22,12 +22,13 @@ use crate::SyntaxKind;
 use crate::SyntaxNode;
 use crate::TreeNode;
 
-/// The name of the `allow_nested_inputs` workflow hint.
-pub const WORKFLOW_HINT_ALLOW_NESTED_INPUTS: &str = "allow_nested_inputs";
+/// The name of the `allow_nested_inputs` workflow metadata hint (i.e.
+/// `allowNestedInputs`) for WDL v1.1 and later.
+pub const WORKFLOW_HINT_ALLOW_NESTED_INPUTS: &str = "allowNestedInputs";
```

Comment:
https://github.com/openwdl/wdl/blob/wdl-1.2/SPEC.md#allow_nested_inputs
Based on my reading, this was correct and now it is reversed.

> allow_nested_inputs
Allowed type: Boolean
Alias: allowNestedInputs

Reply:
The v1.2 spec is a bit revisionist... In WDL v1.1 `allow_nested_inputs` isn't accepted anywhere, and you must use the camelCase form. So while the 1.2 spec swaps them, it would be weird to only accept an "alias" for 1.1 and reject the "non-alias" form

Reply:
I see. I still think it'll be a long-term complication that we have the `WORKFLOW_HINT_ALLOW_NESTED_INPUTS` and `WORKFLOW_HINT_ALLOW_NESTED_INPUTS_ALIAS` swapped compared to the spec going forward. I'd suggest we simply add a comment below where the alias would be used that states something like: pre-1.2 this was the only valid option, but now it is considered the alias.

Reply:
It was made this way intentionally to follow the 1.2 spec and going forward.

`allowNestedInputs` is now the alias and not the preferred name; it may be deprecated in a future version of the spec.